### PR TITLE
gamma added to tensorflow random

### LINKED ISSUE #10864: gamma

### PR DESCRIPTION
**Pull request by ian0549**
close #10864

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py [hunk 1]
```diff
@@ -90,3 +90,41 @@ def test_tensorflow_normal(
...
+@handle_frontend_test(
+    fn_tree="tensorflow.random.gamma",
+    shape=helpers.get_shape(allow_none=False, min_num_dims=1, min_dim_size=1),
+    alpha=helpers.floats(min_value=1, max_value=10),
+    beta=helpers.floats(min_value=1, max_value=10),
+    dtype=helpers.get_dtypes("float", full=True),
+    seed=helpers.ints(min_value=0, max_value=10),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_Gamma(
```
Comment (kurshakuz) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py:105]:
```suggestion
def test_tensorflow_gamma(
```

File: ivy/functional/frontends/tensorflow/random.py [hunk 2]
```diff
@@ -19,3 +19,22 @@ def uniform(shape, minval=0, maxval=None, dtype=ivy.float32, seed=None, name=Non
...
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "float16",
+            "bfloat",
+            "float64",
+        )
+    },
+    "tensorflow",
```
Comment (kurshakuz) [inline ivy/functional/frontends/tensorflow/random.py:36]: Is there any specific reason why all of the above data types are not supported?

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py (hunk 1 above)
Comment (kurshakuz) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py:105]: We use lowercase naming for all functions

**Review by kurshakuz — CHANGES_REQUESTED** [review]
Hi! 

I have tested your implementation locally and it is failing one test. Can you please fix it and let me know once you are done?

```
E           ivy.utils.exceptions.IvyBackendException: torch: gamma: to() received an invalid combination of arguments - got (NoneType, FloatDtype), but expected one of:
E            * (torch.device device, torch.dtype dtype, bool non_blocking, bool copy, *, torch.memory_format memory_format)
E            * (torch.dtype dtype, bool non_blocking, bool copy, *, torch.memory_format memory_format)
E            * (Tensor tensor, bool non_blocking, bool copy, *, torch.memory_format memory_format)
E           
E           Falsifying example: test_tensorflow_Gamma(
E               shape=(1,),
E               alpha=1.0,
E               beta=1.0,
E               dtype=['bfloat16', 'float32'],
E               seed=0,
E               test_flags=FrontendFunctionTestFlags(
E                   num_positional_args=0,
E                   with_out=False,
E                   inplace=False,
E                   as_variable=[False],
E                   native_arrays=[False],
E               ),
E               fn_tree='ivy.functional.frontends.tensorflow.random.gamma',
E               frontend='tensorflow',
E               on_device='cpu',
E           )
E           
E           You can reproduce this example by temporarily adding @reproduce_failure('6.68.2', b'AXicY2RAAYyYbAAAXgAE') as a decorator on your test case

ivy/utils/exceptions.py:128: IvyBackendException
```

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**